webapp/backend/pkg/models/testdata/helper.go

```diff
@@ -32,7 +33,7 @@ func main() {
 		log.Fatalf("ERROR %v", err)
 	}
 	defer file.Close()
-	_, err = SendPostRequest("http://localhost:9090/api/devices/register", file)
+	_, err = SendPostRequest("http://localhost:8080/api/devices/register", file)
```

Comment:
Why change the port for testing?
on mac's 8080 is used by a built in service, so I use 9090 instead.